packages/flutter/lib/src/material/radio_list_tile.dart

```diff
@@ -485,15 +485,11 @@ class RadioListTile<T> extends StatelessWidget {
     }
 
     Widget? leading, trailing;
-    switch (controlAffinity) {
-      case ListTileControlAffinity.leading:
-      case ListTileControlAffinity.platform:
-        leading = control;
-        trailing = secondary;
-      case ListTileControlAffinity.trailing:
-        leading = secondary;
-        trailing = control;
-    }
+    (leading, trailing) = switch (controlAffinity) {
+      ListTileControlAffinity.leading ||
+      ListTileControlAffinity.platform => (control, secondary),
```

Comment:
This looks strange, even though it is going to be a very long line, I would put this all in one.

Reply:
```suggestion
      ListTileControlAffinity.leading || ListTileControlAffinity.platform => (control, secondary),
```